### DISCUSSION TITLE
What happens when a connector crashes in the middle of a sync?

### DISCUSSION THREAD

**Discussion by tzahush** [airbytehq/airbyte#33059] [Connector Questions]
Hey guys, I would like to understand what happens when a connector crashes in the middle, will it know to continue from where it was?

**Comment by sajarin** [comment]
hey @tzahush 

It depends on the connector and the cause of the crash. If the connector is able to checkpoint its state, then you may have what is called a partial sync failure (some records are synced but not all). Most likely though, you'd want to retry the sync anyway in the case of a failure.

[a comment by an automated account is not shown]
